static/app/utils/replayCount/useReplayCountForIssues.tsx

```diff
@@ -14,9 +16,10 @@ interface Props {
  */
 export default function useReplayCountForIssues({
   bufferLimit = 25,
-  statsPeriod = '14d',
```

Comment:
removed this bc of logic from this pr: https://github.com/getsentry/sentry/pull/68396

Reply:
We probably don't need to change defaults, especially to longer values. if something doesn't like the default it should be overridden, or even better is to have no default at all and force call sites to put in what they want.